package.json

```diff
@@ -12,7 +12,7 @@
     "compile": "lerna run compile",
     "hint": "yarn solhint \"./source/**/*.sol\"",
     "lint": "yarn eslint \"./**/*.js\"",
-    "ganache": "ganache-cli -p 8545 --gasLimit 0xfffffffffff --time '2017-05-10T00:00:00+00:00'",
+    "ganache": "ganache-cli -p 8545 --gasLimit 0xfffffffffff --time '2017-05-10T00:00:00+00:00' -s 0",
```

Comment:
why do we need the seed flag?

Reply:
put it in as needed to have a fixed account address in ganache for testing.

Reply:
Essentially you can't pull in `accounts[0]` to the order tests as it isnt a truffle file. Instead I have just copied in the address. It's needed so that we can test ganache signing being provided only the address.